auto_round/export/export_to_itrex/export.py

```diff
@@ -185,15 +185,11 @@ def pack_model(
             5. zeros is always needed even for sym.
         inplace (bool, optional): Compress the model in place, or copy the model and compress it.
 
-    xpu args:
-        compression_dtype=torch.int8,
-        compression_dim=0,
-        use_optimum_format=False,
-        scale_dtype=convert_dtype_str2torch(config.scale_dtype),
-        device="xpu",
     """
+    # Due to XPU doesn't support tuning yet
+    device = detect_device() if device == "xpu" else device
```

Comment:
xpu may be detected as "xpu:0". Please verify this function on xpu device.

Reply:
xpu device will be detected as 'cpu'

Reply:
this is not a good  code style. An workaround is  hardcode "cpu" for xpu in this code. In the future, we should support detect xpu in detect_device

Reply:
ok，will update it to the cpu device